v2/pkg/passive/sources.go

```diff
@@ -93,11 +93,19 @@ var AllSources = [...]subscraping.Source{
 	// &reconcloud.Source{}, // failing due to cloudflare bot protection
 }
 
+var sourceWarnings = map[string]string{
+	"passivetotal": "New API credentials for PassiveTotal can't be generated, but existing user account credentials are still functional. Please ensure your integrations are using valid credentials.",
+}
+
 var NameSourceMap = make(map[string]subscraping.Source, len(AllSources))
 
 func init() {
 	for _, currentSource := range AllSources {
 		NameSourceMap[strings.ToLower(currentSource.Name())] = currentSource
+
+		if warning, ok := sourceWarnings[strings.ToLower(currentSource.Name())]; ok {
```

Comment:
Wouldn't this potentially print the warning two times? In my opinion it makes sense to print the warning in case the source is selected for a search

Reply:
Uggh, forgot to remove it from here /shrug